LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -264,7 +264,10 @@ extension TherapySettingsView {
 
     private var basalRatesSection: Card {
         card(for: .basalRate) {
-            if let items = viewModel.therapySettings.basalRateSchedule?.items, let supportedBasalRates = viewModel.pumpSupportedIncrements()?.basalRates {
+            if let items = viewModel.therapySettings.basalRateSchedule?.items,
+               let supportedBasalRates = viewModel.pumpSupportedIncrements()?.basalRates,
+               let total = viewModel.therapySettings.basalRateSchedule?.total()  {
```

Comment:
This reads somewhat like you do not want to display any part of the card if you can't display the total.  I.e. it suggests to the reader that total() might return nil, but really it's just the schedule that might be nil.
```
if let schedule = viewModel.therapySettings.basalRateSchedule,  let supportedBasalRates = viewModel.pumpSupportedIncrements()?.basalRates {
    let items = schedule.items
    let total = schedule.total()
```

Reply:
"After modifying the basal schedule, seeing the total assists user in verifying no improper entry was made".

If you're trying to address this use case, maybe showing it on the editing screen would be better.  Also, the other use case is rarer occurance (once every 1-6mo), so might be better to have it in the context of the editing screen also, rather than adding cognitive load every time the user goes to settings.